File: Javascript/2419. Longest Subarray With Maximum Bitwise AND.js
var longestSubarray = function(nums) 
{
    let ans = 0, cnt = 0;
    let max_element = Math.max(...nums);
    for(let num of nums)
    {
        if(num == max_element) cnt += 1;
        else cnt = 0;
        ans = Math.max(ans, cnt);
    }
    return ans;
};

File: Python/2419. Longest Subarray With Maximum Bitwise AND.py
class Solution(object):
    def longestSubarray(self, nums):
        
        length = 0
        count = 0
        max_val = max(nums)
        
        for num in nums:
            if num == max_val:
                count += 1
            else:
                count = 0
            length = max(length, count)
        return length
            